source/renderer/app/components/voting/voting-registration-wizard-steps/VotingRegistrationStepsEnterPinCode.js

```diff
@@ -66,12 +67,27 @@ type Props = {
   onSetPinCode: Function,
 };
 
+type State = {
+  selectedPinField: ?string,
+  pinValues: Array<string>,
```

Comment:
Also, the state values are acting in a strange way. If you press backspace a few times, it clears the wrong values and, if you have the cursor after a value, pressing any key will not move to the next input.

![state](https://user-images.githubusercontent.com/1504716/119041256-c8c75a00-b98c-11eb-8318-6ac6c17e2a3a.gif)

Reply:
After analyzing pinValues / repeatedPinValues are not relevant field and i have removed them from state, because data handling is done inside child PinCode component and not on VotingRegistrationStepsEnterPinCode.